doc/level1.rst

```diff
@@ -128,31 +125,32 @@ Accounts controlled by the MUA
 
 A Level 1 MUA maintains an internal structure ``own_state`` for each
 account on which Autocrypt is enabled. ``own_state`` has the following
-members:
+attributes:
 
- * ``secret_key`` -- the RSA secret key material used for
-   the account (see :ref:`secretkeys` ).
- * ``public_key`` -- the OpenPGP transferable public key derived
+ * ``secret_key``: The RSA secret key material used for
+   the account (see :ref:`secretkeys`).
+ * ``public_key``: The OpenPGP transferable public key (:rfc:`OpenPGP
+   "Transferable Public Key"<4880#section-11.1>`) derived
    from the secret key.
- * ``prefer_encrypt`` -- the user's own
-   preferences on the account, either ``mutual`` or ``nopreference``.
-   This SHOULD be set to ``nopreference`` by default.
+ * ``prefer_encrypt``: The user's encryption 
+   preferences for this account.  This is either ``mutual`` or ``nopreference``.
+   This SHOULD default to ``nopreference``.
 
 If Autocrypt is enabled for a given account, the MUA SHOULD allow the
-user to switch the setting for ``own_state.prefer_encrypt``, but this
-choice might normally be hidden in a "preferences pane" or something
-similar.  Please see :ref:`preference-ui` for a specific example of
-how this might look.
+user to switch the setting for ``own_state.prefer_encrypt``.  This
+choice might be hidden in something like a "preferences pane".  See
+:ref:`preference-ui` for a specific example of how this could look.
 
-How this information is managed and used is covered in :ref:`own-state-management`.
+How this information is managed and used is discussed in :ref:`own-state-management`.
 
 .. _peer-state-management:
 
 Peer State Management
 ---------------------
 
-Autocrypt MUAs update state about their communications peers based
-on information gathered from received e-mail headers.
+An Autocrypt MUA updates each communication peer's state using the
```

Comment:
"the state they keep about each communication peer", maybe? the "peer's state" sounds like changing something about the key?

Reply:
ack